src/BaseDAO/CLI.hs

```diff
@@ -192,6 +200,13 @@ serveContractRegistry execName gitRev contracts version =
       PrintMetadata config ->
         putTextLn . decodeUtf8 . encodePretty $
           DAO.knownBaseDAOMetadata (DAO.mkMetadataSettings config)
+      PrintMetadataRegistry config -> let
+        metadataSettings = DAO.MetadataSettings
+          { msConfig = config
+          , mcOperatorsL = sOperatorsL
+          }
+        in putTextLn . decodeUtf8 . encodePretty $
+          (RegistryDAO.knownRegistryDAOMetadata @MText @MText metadataSettings)
```

Comment:
Why not use `DAO.mkMetadataSettings` here as well?
The result should be the same

Reply:
I actually tried it, but couldn't get the types to work, as the `mkMetadataSettings` returns value of type `MetadataSettings (Storage () ())` but here we want it to be of type `MetadataSettings (Storage (RegistryDaoContractExtra k v) (RegistryDaoProposalMetadata k v))`

Reply:
I see, it seems that most functions in `Lorentz.Contracts.BaseDAO.TZIP16Metadata` are polymorphic, but not `mkMetadataSettings`, possibly because it was expected to be the only one.

I think it's worth generalizing it here to be able to produce a `MetadataSettings` for any `store` that satisfies the constraints.

I think it shouldn't be too hard to build the `Lens` there, given that `MetadataSettings` already requires the store to have `"sOperators" := Operator ~> ()`

Reply:
I have included it. Hope it is what you actually wanted.

Reply:
Looks good to me, but it seems that the `config` shadows the existing binding and is causing CI to fail.

Reply:
Have fixed it. But it seems that the contract is oversized now. Trying to fix it.